oak_containers_orchestrator_client/src/lib.rs

```diff
@@ -44,6 +44,8 @@ use self::proto::oak::{
     key_provisioning::v1::GroupKeys,
     session::v1::AttestationEvidence,
 };
+mod orchestrator_client;
+pub use crate::orchestrator_client::OrchestratorClient;
```

Comment:
It looks like this crate does not use the OrchestratorClient, it only exposes it?